docs-source/content/quickstart/create-domain.md

```diff
@@ -102,14 +71,23 @@ in the ***Security*** section.
 
 1.	To confirm that the load balancer noticed the new Ingress and is successfully routing to the domain's server pods,
     you can send a request to the URL for the "WebLogic ReadyApp framework" which will return a HTTP 200 status code, as
-    shown in the example below.  If you used the host-based routing Ingress sample, you will need to
-    provide the hostname in the `-H` option.
+    shown in the example below.  
 
-    Substitute the Node IP address of the worker node for `your.server.com`. You can find it by running:
+    **NOTE**: Substitute the Node IP address of the worker node for `your.server.com`. You can find it by running:
 
     ```bash
-    $ kubectl get po -n sample-domain1-ns -o wide
+    $ kubectl get pods -n sample-domain1-ns -o wide
+    ```
+    Here is an example of the output of this command:
+
+    ```
+    $ kubectl get pods -n sample-domain1-ns -o wide
```

Comment:
Did you find any connection between the output of this command and the address you need to supply? I did not. In my testing, I found that the address to use is simply the address of the box on which kubernetes is running. I used `localhost` on my Mac. That also works when running locally on my hosted box, and so does the host name of that box when running elsewhere (curl -v -H 'host: sample-domain1.org' busgl1108.us.oracle.com:30305/weblogic/ready). I think that the `get pods` command may have been intended for other purpose.

Reply:
I worked with Mark on it. This command displays the Node IP address and that address is supposed to be used in the CURL command to connect to the ReadyApp, `curl -v -H 'host: sample-domain1.org' http://your.server.com:30305/weblogic/ready`
Is that not the case?

Reply:
No; I found absolutely no connection between the IP addresses listed in that command's output, and what you need to use in the CURL command.

Reply:
I will investigate this matter further and get back to you. Thanks.

Reply:
I committed a change to the doc that I hope addresses your concerns satisfactorily. Please let me know if that is not the case.

Reply:
Did you push this new change? I don't see it.